src/Silex/Application/TranslationTrait.php

```diff
@@ -46,6 +48,14 @@ public function trans($id, array $parameters = array(), $domain = 'messages', $l
      */
     public function transChoice($id, $number, array $parameters = array(), $domain = 'messages', $locale = null)
     {
-        return $this['translator']->transChoice($id, $number, $parameters, $domain, $locale);
+        $transChoiceExists = method_exists(Translator::class, 'transChoice');
```

Comment:
does this class exist in specific minors of ^4.4/^5.0? I see we dropped 3.x.x so how come there will be a point where this class does not exist/have the method needed?

Reply:
transChoice exists in older versions but in newer versions this method is dropped.